packages/next/components/listInfo/index.js

```diff
@@ -101,59 +130,57 @@ export default function ListInfo({ space }) {
       </LogoWrapper>
       <AboutWrapper>
         <AboutItem>
-          <AboutIcon src="/imgs/icons/network.svg" />
+          <AboutIcon src="/imgs/icons/threshold.svg" />
           <div>
-            <AboutName>Networks</AboutName>
-            <ChainIcons>
-              {space.networks?.map((network, index) => (
-                <ChainIcon key={index} chainName={network.network} />
-              ))}
-            </ChainIcons>
+            <AboutName onClick={handleShowModal}>Threshold</AboutName>
+            <AboutDetail>
+              <ValueDisplay
+                value={space.proposeThreshold}
+                space={space}
+              ></ValueDisplay>
+            </AboutDetail>
           </div>
-          <Tooltip
-            content={space.networks
-              ?.map((item) => capitalize(item.network))
-              .join(", ")}
-            size="full"
-          >
-            <div />
-          </Tooltip>
         </AboutItem>
         <AboutDivider />
         <AboutItem>
-          <AboutIcon src="/imgs/icons/threshold.svg" />
+          <AboutIcon src="/imgs/icons/strategy.svg" />
           <div>
-            <AboutName>Threshold</AboutName>
-            <AboutDetail>{`${toPrecision(
-              space.proposeThreshold,
-              space.decimals
-            )} ${space.symbol}`}</AboutDetail>
+            <AboutName onClick={handleShowModal}>
+              Strategy({strategyCount})
+            </AboutName>
+            <StrategyAboutDetail>
```

Comment:
please add a html native `title` prop here, so users can hover to see full content

Reply:
also, use a `span` tag here is perefred